### DISCUSSION TITLE
The value of k

### DISCUSSION THREAD

**Discussion by wjq1981** [hahnlab/CAFE5#141] [Q&A] [answered]
The value of k

Hi, when I use -k 2,3,4,5 get 4 results. But there is still some difference between them. I would like to ask, can I tell by that file or that value of the result that it is reasonable to take a certain value for this -k?

**Comment by benfulton** [comment, marked as answer]
No, the value of k only indicates how finely to discretize your gamma curve. It shouldn't matter too much which one you choose. See #184 for an explanation.
